### PR TITLE
[Chronicle-246] Chronicle use case playbook

### PR DESCRIPTION
**Pull request by crestdatasystems**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
Threat hunting use case playbook, reviewed by @anara123 

## Minimum version of Demisto
- [ ] 4.5.0
- [x] 5.0.0
- [ ] 5.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [x] Tests
- [x] Documentation 

## Demisto Partner?
- [x] The title must be in the following format: **[YOUR_PARTNER_ID] short description**

### CODE REVIEW

**Comment by guykeller** [comment]
@crestdatasystems Have you added all your desired changes to this PR? May I review it?

**Comment by crestdatasystems** [comment]
> 
> 
> @crestdatasystems Have you added all your desired changes to this PR? May I review it?

yes @guykeller, we have made all the changes recommended.

**Comment by guykeller** [comment]
Let's wait for approval from @mayagoldb as well

File: Packs/GoogleChronicleBackstory/Scripts/ChronicleDomainIntelligenceSourcesWidgetScript/ChronicleDomainIntelligenceSourcesWidgetScript.yml
Comment (crestdatasystems) [reply Packs/GoogleChronicleBackstory/Scripts/ChronicleDomainIntelligenceSourcesWidgetScript/ChronicleDomainIntelligenceSourcesWidgetScript.yml:5]: @guykeller this is for WidgetScript tagged as "dynamic-section" which doesn't return anything.

File: Packs/GoogleChronicleBackstory/Scripts/ListDeviceEvents/ListDeviceEventsScript.yml
Comment (crestdatasystems) [reply Packs/GoogleChronicleBackstory/Scripts/ListDeviceEvents/ListDeviceEventsScript.yml:7]: @guykeller will make this change

**Review by mayagoldb — APPROVED** [review]

**Comment by ShahafBenYakir** [comment]
@yaron-libman Hi can we prioritize this?

**Comment by ShahafBenYakir** [comment]
This will be done this week (@yaron-libman correct me if I am wrong please)

**Comment by ShahafBenYakir** [comment]
Hi @crestdatasystems, went over this PR and saw that in the initial commit names of configuration param names were quite confused with additional info that is sometimes needed. 
**In general instead of doing this:**
content  ~:dev:demisto:content  -  :Packs:GoogleChronicleBackstory:Integrations:GoogleChronicleBackstory:GoogleChronicleBacks… 2020-09-30 16-33-03
**Which appears like this in the UI:**
Settings 2020-09-30 16-14-41
**We strongly suggest doing this:**
content  ~:dev:demisto:content  -  :Packs:GoogleChronicleBackstory:Integrations:GoogleChronicleBackstory:GoogleChronicleBacks… 2020-09-30 16-12-41
**Which appears like this in th UI:**
Settings 2020-09-30 16-11-01
**Such that when clicking the question mark a tooltip appears:** 
Settings 2020-09-30 16-11-17

This is not mandatory for now, only if you have the time and want to put the effort.

**Comment by yaron-libman** [comment]
Reviewed and updated the yamls. Please update the relevant md files.

**Comment by ShahafBenYakir** [comment]
@guykeller Let's push this forward

**Comment by crestdatasystems** [comment]
@guykeller I see that the "Run Validations" check has failed. Please let us know if we need to take any action to resolve this.

**Comment by guykeller** [comment]
@crestdatasystems Fixed all the errors, except one, in which I need your cooperation.
There seem to be some added files missing in the release notes, as you can see in the build.
Could you please add them?
Alternatively, you can send me the release notes for the files and I'll add them.

**Comment by crestdatasystems** [comment]
> @crestdatasystems Fixed all the errors, except one, in which I need your cooperation.
> There seem to be some added files missing in the release notes, as you can see in the build.
> Could you please add them?
> Alternatively, you can send me the release notes for the files and I'll add them.

@guykeller We have added the missing release notes.

**Comment by guykeller** [comment]
@crestdatasystems 
All good to go - you just need to sign our license agreement.
Press the `details` button on the `license/cla` step and sign it please.

**Pull request merged**
